crates/store/src/types/application.rs

```diff
@@ -11,8 +11,8 @@ pub struct ApplicationMeta {
     pub blob: key::BlobMeta,
     pub version: Option<Box<str>>,
     pub source: Box<str>,
-    // contract_app_id - Application ID is hash(owner,name) saved in package manager contract
-    pub contract_app_id: Option<Box<str>>,
+    // metadata - Contains application ID is hash(owner,name) saved in package manager contract
+    pub metadata: Option<Box<[u8]>>,
```

Comment:
can just be `Box<[u8]>`, an empty array can indicate a lack of metadata

also, no need for the comment

Reply:
Yes, updated